jib-core/src/main/java/com/google/cloud/tools/jib/api/JavaContainerBuilder.java

```diff
@@ -679,7 +679,16 @@ private JavaContainerBuilder addDirectory(
     return this;
   }
 
-  private void addFileToLayer(
+  /**
+   * Adds a file to a {@link FileEntriesLayer.Builder}.
+   *
+   * @param layerBuilders map that keeps track of all layers, with the key being the layer type and
+   *     value being a {@link FileEntriesLayer.Builder}.
+   * @param layerType the layer type.
+   * @param sourceFile the source file path.
+   * @param pathInContainer path in container.
+   */
+  public static void addFileToLayer(
```

Comment:
Ditto.

We need to think twice before making this part of the public API surface.